lib/Payable/PayableBuilder.php

```diff
@@ -15,4 +15,20 @@ private function buildPayable($payableData)
 
         return new Payable(get_object_vars($payableData));
     }
+
+    /**
```

Comment:
`$payablesData` should be an array of `stdClass`.

The return is an array of payables, that's why I changed to `Payable[]`.